hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -398,7 +398,9 @@ public void init(@NonNull final Platform platform, @NonNull final NodeId nodeId)
             notifications.register(PlatformStatusChangeListener.class, notification -> {
                 switch (notification.getNewStatus()) {
                     case ACTIVE -> {
-                        run();
+                        // This is also called from ServicesMain#run() when the node is restarted.
+                        // Need to confirm if this is needed
+                        //                        run();
```

Comment:
Not a blocker, but `FUTURE` or `TODO` would be a helpful reminder

Reply:
yes agreed. Will add a TODO